src/components/dapp-staking/modals/RegisterDappDescription.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <q-tabs v-model="tab" dense narrow-indicator>
+      <q-tab name="markdown" label="Markdown"></q-tab>
+      <q-tab name="preview" label="Preview"></q-tab>
+    </q-tabs>
+
+    <q-tab-panels v-model="tab">
+      <q-tab-panel name="markdown">
+        <q-scroll-area class="tw-h-96">
+          <q-input
```

Comment:
the text input should be a multi-line text area and stretch it all the way to the bottom of the modal.
![Screen Shot 2021-12-14 at 4 18 17 PM](https://user-images.githubusercontent.com/40356749/146026465-ce25a8ff-c0e8-4a2a-8cfc-ee01c60a490e.png)

Reply:
At the moment text area stretches down as user types. I will try to initially expand it to the bottom.